packages/wallets/src/components/WalletListCardTitle/WalletListCardTitle.tsx

```diff
@@ -3,13 +3,13 @@ import { useWalletAccountsList } from '@deriv/api';
 import { WalletText } from '../Base';
 
 type TProps = {
-    currency: NonNullable<ReturnType<typeof useWalletAccountsList>['data']>[0]['currency'];
+    title: NonNullable<ReturnType<typeof useWalletAccountsList>['data']>[0]['currency'];
```

Comment:
PR for types in root is merged ady, we can use it to type this

Reply:
I will do this in the next PR